config/locales/en.yml

```diff
@@ -1,8 +1,210 @@
+      # divider between a range of dates, such as "2012 to 2014"
+      range_divider: to
+      # hidden label for date filter dropdown
+      show_filters: Show Date Filters
+      # latest date in range day
+      to_day: To Day
+      # latest date in range month
+      to_month: To Month
+      # latest date in range year
+      to_year: To Year
+    help:
+      # section title
+      title: Search Help
+      # search help html, this may need to be recreated per language
+      # instead of translated literally due to differences with stemming and grammar
+      html: |-
+        <h4>Basic Search</h4>
+        <p>Simply enter the word you wish to find and the search engine will search for every instance of the word in the journals. For example: Fight. All instances of the use of the word fight will show up on the results page.</p>
+        <h4>Wildcard Searches</h4>
+        <p>Using an asterisk (*) will increase the odds of finding the results you are seeking. For example: Fight*. The search results will display every instance of fight, fights, fighting, etc. More than one wildcard may be used. For example: *ricar*. This search will return most references to the Aricara tribe, including Ricara, Ricares, Aricaris, Ricaries, Ricaree, Ricareis, and Ricarra. Using a question mark (?) instead of an asterisk (*) will allow you to search for a single character. For example, r?n will find all instances of ran and run, but will not find rain or ruin.</p>
+        <h4>Capitalization</h4>
+        <p>Searches are not case sensitive. For example: george will come up with the same results as George.</p>
+        <h4>Phrase Search</h4>
+        <p>Searching for a specific phrase may help narrow down the results. Rather long phrases are no problem. For example: "This white pudding we all esteem".</p>
+        <h4>Subsequent Searches</h4>
+        <p>Because of the creative spellings used by the journalists, it may be necessary to try your search multiple times. For example: P?ro*. This search brings up numerous variant spellings of the French word pirogue, "a large dugout canoe or open boat." Searching for P?*r*og?* will bring up other variant spellings. Searching for canoe or boat also may be helpful.</p>
+    filters:
+      # clearing filters (not the text search)
+      clear_all: Clear Filters
+      # the word "filter" singular
+      filter: Filter
+      # label above the filters
+      filter_by: Filter by
+      # display when a specific filter option is an empty string
+      no_label: No Label
+      # link text to view all the possible filter options
+      see_more: See more
+      # hidden label to expand a filter category
+      show: "Show %{label} Filters"
+    # placeholder for the text search field
+    placeholder: Search for...
+    results:
+      # how to display number of results when no search / filters have been applied
+      # and therefore user is browsing all items
+      general:
+        zero: 0 results
```

Comment:
I'm guessing `pluralize` isn't i18ned? :)

Reply:
Unfortunately, it did not appear to be, but they do have this fancy option picker based on "zero" "one" and "other," plus it got extra tricky for other languages that use different number pluralization types, haha.

For reference:  http://guides.rubyonrails.org/i18n.html#pluralization

Reply:
Whoa. I like "(paucal)" from the Unicode page on plural forms hehe.
It mentions that the `zero` key is optional, so if that is the same as the plural, we can probably drop it.

Reply:
Is it optional in most of the languages we might be using?  Spanish / German / French?  Since the english one is basically the template I think it would be good to leave it as obvious / encompassing as possible for the sake of the translators who may come.

I realize I'm probably overengineering this since Family Letters might literally be the only project that ever uses this, but I'm paranoid, I guess :)

Reply:
Okay. I thought about suggesting just leaving a commented example that uses it but then dropping the redundancy in the uncommented code. I don't know if any of those languages pluralize differently for zero or not, so making it apparent it's available isn't a bad thing.